src/app/public/modules/toast/types/toast-container-options.ts

```diff
@@ -4,6 +4,7 @@ import {
 
 /**
  * Global configuration options for displaying toast components.
+ * @internal
```

Comment:
This is exported by the `public_api` 🤔

Reply:
This wasn't documented in the current docs. I did a quick search in ADO and found the only place its referenced is the omnibar SPA: https://blackbaud.visualstudio.com/Products/_git/skyux-spa-omnibar?path=%2Fsrc%2Fapp%2Fapp-sky.module.ts&version=GBmaster&line=76&lineEnd=77&lineStartColumn=1&lineEndColumn=1&lineStyle=plain
Seems like it would be OK to not document, but keep in our api for now?